### PR TITLE
Move HTMLtoJSX page off React site

### PR DESCRIPTION
**Pull request by Daniel15**
As per @zpao's request at https://github.com/reactjs/react-magic/issues/52, this moves the HTML to JSX page off the React site and onto the React-Magic site. The new page is at http://magic.reactjs.net/htmltojsx.htm. Currently it hotlinks some files from the React site (stylesheet and live code editor) but I'm planning on making it properly standalone soon.

I've made the old page (`/html-jsx.html`) a "redirect" to the new page. I use the term "redirect" loosely here - Unfortunately Github Pages lacks the ability to do 301 redirects (along with many other basic web hosting features), so I needed to use a meta refresh tag to do the redirect.

### CODE REVIEW

File: docs/html-jsx.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML to JSX</title>
+  <!-- Github Pages lacks basic functionality such as permanent server-side
+       redirects, so the best we can do is a meta redirect -->
+  <meta http-equiv="refresh" content="0; url=http://magic.reactjs.net/htmltojsx.htm" />
```
Comment (zpao) [inline docs/html-jsx.html]: I'm going to merge and then follow up to use the `redirect` layout we have set up for this.

**Comment by zpao** [comment]
Thanks!

File: docs/html-jsx.html (hunk 1 above)
Comment (zpao) [inline docs/html-jsx.html]: PS `.htm`? I think that extension lost the war 😛

Comment (Daniel15) [inline docs/html-jsx.html]: Ahh, I didn't even notice there was a `redirect` layout.

I like `.htm`... It's one less character, and consistent with pretty much everything else having a three-letter file extension (well, apart from `.js`). The extra `l` doesn't provide any extra value  :smile:  haha

Comment (Daniel15) [inline docs/html-jsx.html]: I dream of a world where Github Pages supports server-side redirects... Maybe one day. Missing functionality is the main reason I'm not really using Github Pages for anything much.

**Comment by karanthakral** [comment]
seems the links is not working

**Comment by Daniel15** [comment]
I guess something broke the page. I don't have time to look into it at the
moment, but I'm happy to take a pull request that fixes it.

**Pull request merged**